tests/test_pex_builder.py

```diff
@@ -175,15 +175,17 @@ def build_and_check(path, copy_mode):
             pb.clone(into=path_clone)
 
             for root in path, path_clone:
-                if copy_mode == CopyMode.SYMLINK:
-                    assert os.path.islink(os.path.join(root, "exe.py"))
-                    continue
                 s1 = os.stat(src)
                 s2 = os.stat(os.path.join(root, "exe.py"))
                 is_link = (s1[stat.ST_INO], s1[stat.ST_DEV]) == (s2[stat.ST_INO], s2[stat.ST_DEV])
                 if copy_mode == CopyMode.COPY:
                     assert not is_link
-                elif copy_mode == CopyMode.LINK:
+                else:
+                    # Since os.stat follows symlinks; so in CopyMode.SYMLINK, this just proves the
```

Comment:
Just noticed this language is bad. I think it's grokable though so I'd like to not kill a tree.

Reply:
Yeah, maybe fix in a later PR? Or, if the CI hasn't gone far yet, cancel it and push again.